packages/core/package.json

```diff
@@ -33,6 +33,10 @@
       "types": "./dist/plugins/*.d.ts",
       "default": "./dist/plugins/*.js"
     },
+    "./base/*": {
```

Comment:
After extracting these codes, it seems that the number of lines of code and complexity are even higher.

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/9585c0b5-6b89-470d-9168-c266d6461425)

Reply:
Type calculation is more complex, subsequent processing 😢